noir/noir-repo/acvm-repo/acvm/src/pwg/mod.rs

```diff
@@ -556,6 +566,21 @@ fn any_witness_from_expression(expr: &Expression) -> Option<Witness> {
     }
 }
 
+/// Returns `true` if the predicate is zero
+/// A predicate is used to indicate whether we should skip a certain operation.
+/// If we have a zero predicate it means the operation should be skipped.
+pub fn is_predicate_false(
```

Comment:
```suggestion
pub(crate) fn is_predicate_false(
```

Reply:
surprised clippy didn't complain about this